plugin/pkg/admission/resourcequota/admission_test.go

```diff
@@ -99,15 +101,19 @@ func createHandlerWithConfig(kubeClient kubernetes.Interface, informerFactory in
 	}
 	quotaConfiguration := install.NewQuotaConfigurationForAdmission()
 
-	handler, err := resourcequota.NewResourceQuota(config, 5, stopCh)
+	handler, err := resourcequota.NewResourceQuota(config, 5)
 	if err != nil {
 		return nil, err
 	}
-	handler.SetExternalKubeClientSet(kubeClient)
-	handler.SetExternalKubeInformerFactory(informerFactory)
-	handler.SetQuotaConfiguration(quotaConfiguration)
 
-	return handler, nil
+	initializers := admission.PluginInitializers{}
+	initializers = append(initializers, genericadmissioninitializer.New(kubeClient, informerFactory, nil, nil, stopCh))
+	initializers = append(initializers, kubeapiserveradmission.NewPluginInitializer(nil, nil, quotaConfiguration))
```

Comment:
non-blocking, but if you want, could simplify to

```go
initializers := admission.PluginInitializers{
  genericadmissioninitializer.New(...
  kubeapiserveradmission.NewPluginInitializer(...
}
```

Reply:
fixed